workers/unity/Packages/io.improbable.worker.sdk/EventTracing.cs

```diff
@@ -0,0 +1,425 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Improbable.Worker.CInterop.Internal;
+
+namespace Improbable.Worker.CInterop
+{
+    public struct SpanId : IEquatable<SpanId>
+    {
+        internal static UIntPtr SpanIdSize = new UIntPtr(16);
+        public unsafe fixed byte Data[16];
+
+        public static SpanId NullSpanId => GetNullSpanId();
```

Comment:
```suggestion
        public static SpanId Null => GetNullSpanId();
```

`SpanId.Null` is just as descriptive as `SpanId.NullSpanId`

Reply:
Updated in latest commit